### PR TITLE
2fa wording and descriptions have been updated

### PR DESCRIPTION
**Pull request by SpyrosChr**

### CODE REVIEW

File: 2fa/v1/openapi.yaml [hunk 1]
```diff
@@ -19,10 +23,17 @@ security:
...
 tags:
-  - name: 2fa-application-service
-    description: 2FA Application Service
-  - name: 2fa-otp-service
-    description: 2fa Otp Service
+  - name: "Application Service"
+    description: |
+      The 2FA API has a part where the user can create different 2FA applications and separate his configuration.
+      tyntec stores these configurations for the user and the user can refer to the applicationId that points to 
+      the desired configuration that will be applied for the specific OTP delivery.
```
Comment (coders-kitchen) [inline 2fa/v1/openapi.yaml]: What is an OTP? Might makes sense to write it On Time Password (OTP)

File: 2fa/v1/openapi.yaml [hunk 2]
```diff
@@ -19,10 +23,17 @@ security:
...
+
+      There is a default application for each user which contains some default values and an English language template.
+      This default application will be used in case no applicationId is specified in an OTP delivery request.
+      The user can change the default parameters of the default application as well.
+  - name: "OTP Service"
```
Comment (coders-kitchen) [inline 2fa/v1/openapi.yaml]: This is somehow confusing when the section talks about 2FA, or?

File: 2fa/v1/openapi.yaml [hunk 3]
```diff
@@ -65,9 +76,13 @@ paths:
             $ref: '#/definitions/ErrorResponse'
     post:
       tags:
-        - 2fa-application-service
-      summary: Creates a 2fa application
-      description: Creates an application instance. Values that are not defined will be populated with default values.
+        - "Application Service"
+      summary: Create 2fa application
+      description: |
+        Creates a 2fa application instance. You can either:
```
Comment (coders-kitchen) [inline 2fa/v1/openapi.yaml]: new line needed in order to have the items processed as expected.

File: 2fa/v1/openapi.yaml [hunk 4]
```diff
@@ -111,8 +126,9 @@ paths:
   '/2fa/v1/application/{applicationId}':
     get:
       tags:
-        - 2fa-application-service
-      summary: Get a 2fa application
+        - "Application Service"
+      summary: Get 2fa application
+      description: You can get a specific application by using the GET HTTP method and the application’s UUID
```
Comment (coders-kitchen) [inline 2fa/v1/openapi.yaml]: I would write it as 

    Returns the specified application.

The GET is clear as well the format of the applicationId should be stated somehwere.

File: 2fa/v1/openapi.yaml [hunk 5]
```diff
@@ -121,7 +137,7 @@ paths:
       parameters:
         - name: applicationId
           in: path
-          description: applicationId
+          description: applicationId to be returned
           required: true
           type: string
```
Comment (coders-kitchen) [inline 2fa/v1/openapi.yaml]: add here "format: uuid"

**Comment by coders-kitchen** [comment]
One general note. I would recommend not repeating the information that is already in the definition. 

Sample would be the repetition that applicationid is part of the path etc. pp. .

And check what can be expressed by using default values or format instructions. This would allow as well better auto generation of code samples.

File: 2fa/v1/openapi.yaml (hunk 1 above)
Comment (SpyrosChr) [reply 2fa/v1/openapi.yaml]: OK, i will add this.

File: 2fa/v1/openapi.yaml (hunk 2 above)
Comment (SpyrosChr) [reply 2fa/v1/openapi.yaml]: Yes, it was a little bit confusing. I rephrase those paragraphs. I guess now it is better and more understandable.

File: 2fa/v1/openapi.yaml (hunk 3 above)
Comment (SpyrosChr) [reply 2fa/v1/openapi.yaml]: OK done!

File: 2fa/v1/openapi.yaml (hunk 4 above)
Comment (SpyrosChr) [reply 2fa/v1/openapi.yaml]: It is too long. I will use something shorter.
The application ID should be described in the table below as you said in your next comment.

File: 2fa/v1/openapi.yaml (hunk 5 above)
Comment (SpyrosChr) [reply 2fa/v1/openapi.yaml]: added on the type

**Pull request merged**